module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationKeyCredential.md

```diff
@@ -8,62 +19,103 @@ schema: 2.0.0
 # New-EntraBetaApplicationKeyCredential
 
 ## Synopsis
+
 Creates a key credential for an application.
 
 ## Syntax
 
-```
-New-EntraBetaApplicationKeyCredential [-Usage <KeyUsage>] -ObjectId <String> [-Type <KeyType>]
- [-EndDate <DateTime>] [-Value <String>] [-StartDate <DateTime>] [-CustomKeyIdentifier <String>][<CommonParameters>]
+```powershell
+New-EntraBetaApplicationKeyCredential 
+ -ObjectId <String>
+ [-CustomKeyIdentifier <String>] 
+ [-Type <KeyType>] 
+ [-Usage <KeyUsage>]
+ [-Value <String>] 
+ [-EndDate <DateTime>] 
+ [-StartDate <DateTime>]
+ [<CommonParameters>]
 ```
 
 ## Description
-The New-EntraBetaApplicationKeyCredential cmdlet creates a key credential for an application.
+
+The `New-EntraBetaApplicationKeyCredential` cmdlet creates a key credential for an application.
+
+An application can use this command along with `Remove-EntraBetaApplicationKeyCredential` to automate the rolling of its expiring keys.
+
+As part of the request validation, proof of possession of an existing key is verified before the action can be performed.
 
 ## Examples
 
 ### Example 1: Create a new application key credential
+
+```powershell
+Connect-Entra -Scopes 'Application.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Application.ReadWrite.OwnedBy' #Application Permission
+
+$AppId = (Get-EntraApplication -Top 1).Objectid
+$params = @{
+    ObjectId = $AppId
+    CustomKeyIdentifier = 'EntraPowerShellKey'
+    StartDate = '11/7/2016'
+    Type = 'Symmetric'
+    Usage = 'Sign'
+    Value = '<my-value>'
+}
+
+New-EntraBetaApplicationKeyCredential @params
 ```
-PS C:\> $AppID = (Get-EntraBetaApplication -Top 1).Objectid
-PS C:\> New-EntraBetaApplicationKeyCredential -ObjectId $AppId -CustomKeyIdentifier "Test" -StartDate "11/7/2016" -Type "Symmetric" -Usage "Sign" -Value "123"
 
+```Output
 CustomKeyIdentifier : {84, 101, 115, 116}
 EndDate             : 11/7/2017 12:00:00 AM
-KeyId               : a5845538-3f67-402d-a03e-36d768f1441e
+KeyId               : aaaaaaaa-0b0b-1c1c-2d2d-333333333333
 StartDate           : 11/7/2016 12:00:00 AM
 Type                : Symmetric
 Usage               : Sign
 Value               : {49, 50, 51}
 ```
 
-The first command gets the ID of an application by using the Get-EntraBetaApplication (./Get-EntraBetaApplication.md)cmdlet.
-The command stores it in the $AppId variable.
+This example shows how to create an application key credential.
```

Comment:
In example section we need to add decription for Prarameters used in cmdlets  ObjectId,  CustomKeyIdentifier 
    StartDate .    Type,    Usage ,    Value